<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>421</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Выбор теста</string>
       </property>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Тонкое</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Традиционное</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Вид пиццы</string>
       </property>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Маргарита</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Пеперонни</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Морская</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_6">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>50</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>4 сыра</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Оливки</string>
       </property>
       <widget class="QRadioButton" name="radioButton_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Да</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_8">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Нет</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Пепперони</string>
       </property>
       <widget class="QRadioButton" name="radioButton_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Да</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_10">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Нет</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Двойной сыр </string>
       </property>
       <widget class="QRadioButton" name="radioButton_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Да</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_12">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Нет</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Ананасы</string>
       </property>
       <widget class="QRadioButton" name="radioButton_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Да</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_14">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Нет</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>497</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Pizza!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
